requirements-pants.txt

```diff
@@ -0,0 +1,100 @@
+# Add/remove direct 3rd party dependencies here, with version constraints if necessary.
+# Then run `./pants generate-lockfiles --resolve=st2` to regenerate the lockfile.
```

Comment:
Is this relevant?

Reply:
So, the `generate-lockfiles` suggestion is not useful yet because I haven't finished wiring up the resolves and stuff. I suppose I can remove this line and re-add it once generating lockfiles works.

Reply:
I deleted the `generate-lockfiles` lines for now, since they won't work yet.